ERROR:root:Error occured: could not convert string to float: 'GV'
25-May-22 09:21:16 - Error occured: could not convert string to float: 'GV'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 335, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'GV'
25-May-22 11:20:06 - Error occured: could not convert string to float: 'GV'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 328, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'GV'
25-May-22 11:23:26 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 328, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:23:26 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 328, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:24:16 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 330, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:24:16 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 330, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:24:41 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:24:41 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:29:15 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:29:15 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:29:49 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:29:49 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:30:08 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:30:08 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:30:35 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:30:35 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:30:50 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:30:50 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:31:04 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:31:04 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:31:23 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:31:23 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:31:34 - Error occured: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 115, in min_max_weighting_getter
    min_port_weighting = int(input("Enter minimum desired portfolio weighting: "))
  File "C:\Users\RArm1\Anaconda3\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
25-May-22 11:31:36 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:31:36 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
25-May-22 11:35:39 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
25-May-22 11:35:39 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 13:40:41 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 13:40:41 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 13:54:41 - Error occured: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 115, in min_max_weighting_getter
    min_port_weighting = int(input("Enter minimum desired portfolio weighting: "))
  File "C:\Users\RArm1\Anaconda3\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
26-May-22 13:59:52 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 13:59:52 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 14:02:26 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 14:02:26 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 14:06:57 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 14:06:57 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 14:08:33 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 14:08:33 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 14:08:43 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 14:08:43 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
26-May-22 14:10:24 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
26-May-22 14:10:24 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
27-May-22 09:09:17 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
27-May-22 09:09:17 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
27-May-22 09:10:28 - Error occured: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 115, in min_max_weighting_getter
    min_port_weighting = int(input("Enter minimum desired portfolio weighting: "))
  File "C:\Users\RArm1\Anaconda3\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
27-May-22 12:40:05 - Error occured: could not convert string to float: 'VG Score'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'VG Score'
27-May-22 12:40:05 - Error occured: could not convert string to float: 'V_G_PLACEHOLDER'
Traceback (most recent call last):
  File "C:\Users\RArm1\PycharmProjects\Portfolio Optimiser\Portfolio_Optimiser.py", line 332, in find_vg_score
    test_list.append(float(i[3]))
ValueError: could not convert string to float: 'V_G_PLACEHOLDER'
